src/hw1/Main.java

```diff
@@ -0,0 +1,325 @@
+        this.date = date;
+        this.startTime = startTime;
+        this.endTime = endTime;
+    }
+}
+
+public class Main {
+
+    static Scanner scanner = new Scanner(System.in);
+    static ArrayList<CoworkingSpace> spaces = new ArrayList<>(); // list of coworking spaces
+    static ArrayList<Reservations> reservations = new ArrayList<>(); // list of reservations
+    static int reservationIdCounter = 1; // generate a unique reservation ID for each reservation
+
+    public static void main(String[] args) {
+
+        spaces.add(new CoworkingSpace(1, "Open Space", 1500.0, true));
+        spaces.add(new CoworkingSpace(2, "Private Room", 2500.0, true));
+        spaces.add(new CoworkingSpace(3, "Meeting Room", 3000.0, true));
+
+        boolean exit = false;
+
+        while (!exit) {
+            System.out.println("<< Welcome to a Coworking Space Reservation System >>");
+            System.out.println("1. Admin Login");
+            System.out.println("2. User Login");
+            System.out.println("3. Exit");
+            System.out.println("Enter your choice: ");
+
+            int choice = scanner.nextInt();
+
+            switch (choice) {
+                case 1:
+                    adminMenu();
+                    break;
+                case 2:
+                    customerMenu();
+                    break;
+                case 3:
+                    exit = true;
+                    System.out.println("Thanks for using our service!");
+                    break;
+                default:
+                    System.out.println("Invalid choice!");
+            }
+        }
+    }
+
+
+
+/*
+* Admin Menu: This method is intended to add a new coworking space,
+* remove a coworking space, or view all reservations.
+* Additionally, we can return to the main menu without exiting.
+*/
+    private static void adminMenu() {
+
+        boolean backToMain = false;
+
+        while (!backToMain) {
+            System.out.println("<< Admin Menu >>");
+            System.out.println("1. Add a new coworking space");
+            System.out.println("2. Remove a coworking space");
+            System.out.println("3. View all reservations");
+            System.out.println("4. Back to Main Menu");
+            System.out.println("Enter your choice: ");
+
+            int choice = scanner.nextInt();
+
+            switch (choice) {
+                case 1:
+                    addSpace();
+                    break;
+                case 2:
+                    removeSpace();
+                    break;
+                case 3:
+                    viewReservations();
+                    break;
+                case 4:
+                    backToMain = true;
+                    System.out.println("You're back to the main menu!");
+                    break;
+                default:
+                    System.out.println("Invalid choice!");
+            }
+        }
+    }
+
+    private static void addSpace() {
```

Comment:
try to avoid using everywhere static methods, ex. it could be under some AdminService class and you can call that method by creating an instance of the class (AdminService adminService = new AdminService() and adminService.addSpace() )